### DISCUSSION TITLE
How  do you finish a campaign

### DISCUSSION THREAD

**Discussion by peteauau** [Khopa/dcs_liberation#1921] [Q&A] [answered]
as post  suggest?

**Comment by Starfire13** [comment]
By capturing all the control points.

**Reply by peteauau** [reply]
> By capturing all the control points.

where  do i find  the  control points

**Comment by Starfire13** [comment]
Though you can just bomb all the runways too. If OPFOR can no longer field any aircraft, you win.

**Comment by peteauau** [comment]
Ok will  try  your  suggestions  being  doing that  for the  past 4  missions  now  sending  out  all my  aircraft  for bombing   the  aircraft and  runways.  For  the  control point  my   fwd   marker    there  are  only  about  6    red  units  left  where  i  got  around  over  a  hundred  but  they  will not  capture   the  town,  how  do i make  them  capture  i have tried  elim  etc  and  all the  rest  but  no avail  my  team  will not  advance  further  that  they are  now.

**Comment by Starfire13** [comment, marked as answer]
That info is in the wiki.
https://github.com/dcs-liberation/dcs_liberation/wiki/Base-Capture
